### PR TITLE
Fixing update ci files workflow

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

File: templates/github/.github/workflows/update_ci.yml.j2 [hunk 1]
```diff
@@ -34,12 +34,12 @@ jobs:
         with:
           committer: {{  'pulpbot <pulp-infra@redhat.com>' }}
           author: {{  'pulpbot <pulp-infra@redhat.com>' }}
-          branch: ${GITHUB_REF#refs/heads/}-update-ci-files
-          base: ${GITHUB_REF#refs/heads/}
-          title: 'Update CI for ${GITHUB_REF#refs/heads/}'
+          branch: {{ plugin_default_branch }}-update-ci-files
+          base: {{ plugin_default_branch }}
+          title: 'Update CI files'
```
Comment (mdellweg) [inline templates/github/.github/workflows/update_ci.yml.j2:39]: Was this supposed to update the files on release branches too?

**Review by mdellweg — APPROVED** [review]

File: templates/github/.github/workflows/update_ci.yml.j2 (hunk 1 above)
Comment (daviddavis) [reply templates/github/.github/workflows/update_ci.yml.j2:39]: Eventually I am hoping to handle that. But for now, it just handles `plugin_default_branch`.

Comment (daviddavis) [reply templates/github/.github/workflows/update_ci.yml.j2:39]: Ug yes, good catch. I suppose I need to find a way to handle that. The problem is that `${GITHUB_REF#refs/heads/}` doesn't resolve here. I suppose I need to ditch this `peter-evans/create-pull-request@v3` and write a script to create a PR.

Comment (fao89) [reply templates/github/.github/workflows/update_ci.yml.j2:39]: I don't think you need to ditch it, it works here:
https://github.com/pulp/plugin_template/blob/master/templates/github/.github/workflows/create-branch.yml.j2#L52-L74
I believe you need to create the branch first, and 
```
git checkout -b $BASE_BRANCH-update-ci-files
echo "BASE_BRANCH=some logic before" >> $GITHUB_ENV
```

File: templates/github/.github/workflows/update_ci.yml.j2 [hunk 2]
```diff
@@ -34,12 +34,10 @@ jobs:
         with:
           committer: {{  'pulpbot <pulp-infra@redhat.com>' }}
           author: {{  'pulpbot <pulp-infra@redhat.com>' }}
-          branch: ${GITHUB_REF#refs/heads/}-update-ci-files
-          base: ${GITHUB_REF#refs/heads/}
```
Comment (daviddavis) [inline templates/github/.github/workflows/update_ci.yml.j2:38]: I read through [the docs](https://github.com/peter-evans/create-pull-request#action-inputs) and it looks like if I omit the inputs it'll do what I want by automatically creating a new branch and using the workflow's base branch.

**Comment by daviddavis** [comment]
Looks like the main problem now is that the RELEASE_TOKEN secret doesn't have enough permissions to edit the workflow files.

Will follow up with @RCMariko next week to get a new token with the needed permissions.

**Review by dralley — APPROVED** [review]

**Comment by daviddavis** [comment]
I'm going to go ahead and merge this so it will propagate to repos and then I will fix the token next week.

**Pull request merged**